develop/reference_sql.rst

```diff
@@ -12,7 +12,48 @@ In the following sections, we discuss the different types of queries you can run
 Aggregate Functions
 -------------------
 
-Citus supports and parallelizes most aggregate functions supported by PostgreSQL. Citus's query planner transforms the aggregate into its commutative and associative form so it can be parallelized. In this process, the workers run an aggregation query on the shards and the coordinator then combines the results from the workers to produce the final output.
+Citus supports and parallelizes most aggregate functions supported by
+PostgreSQL. Citus's query planner transforms the aggregate into its commutative
+and associative form so it can be parallelized. In this process, the workers
+run an aggregation query on the shards and the coordinator then combines the
+results from the workers to produce the final output.
+
+Aggregate execution works in two ways:
+
+1. When the aggregate is grouped by a table's distribution column, and there
```

Comment:
> and there are no aggregates on distinct values of non-distribution columns,

This can be removed. When grouping by the distribution column, aggregates can always be pushed down